CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+## Contributing
+
+We do not allow contributors to claim issues. If you find something interesting you can contribute to the repo, feel free to raise a PR
+
+1. Fork the repo
+2. Install dependencies locally `yarn`
+3. Make sure it builds `yarn start`
+4. Make your changes locally
+5. Ensure the code coverage is the same or higher than before your changes.
+6. Create a PR to the `master` branch
```

Comment:
Numbered lists in markdown should all begin with "1." the post processor will auto increment the list when displayed. You can verify this by checking how it renders in your branch https://github.com/salilbc/design-systems-cli/blob/master/CONTRIBUTING.md

Reply:
The reason for doing this is, in the future, if you want to add a new step at point 3, then you don't need to re-number all the following points as github will do it for you.

Reply:
Made the change